homeassistant/components/media_player/yamaha_musiccast.py

```diff
@@ -47,16 +49,48 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Yamaha MusicCast platform."""
+    import socket
     import pymusiccast
 
+    known_hosts = hass.data.get(KNOWN_HOSTS_KEY)
+    if known_hosts is None:
+        known_hosts = hass.data[KNOWN_HOSTS_KEY] = []
```

Comment:
`know_hosts = hass.data.get() or []`

Reply:
@pvizeli thanks for you comment, but I disagree here:

.get() expects an argument. We could use hass.data.get(KNOWN_HOSTS_KEY, []) instead.

But then again line 57 sets known_hosts AND hass.data[KNOWN_HOSTS_KEY] = [].